packages/redux-routine/src/test/index.js

```diff
@@ -143,4 +143,21 @@ describe( 'createMiddleware', () => {
 
 		expect( store.getState() ).toBe( 2 );
 	} );
+
+	it( 'does not recurse when action like object returns from a sync ' +
+		'control', () => {
```

Comment:
Is the concatenation here something automated? Or are you especially strict with line length? 😄 

I don't have any strong feelings on it, but it made it slightly more difficult for me to read the test case (or at least, it came as a surprise to me in reading the label).

Reply:
Of the various options I've played around with for handling line length issues, this is the one I've landed on as the one I can be most consistent with and also indents nicely.

Reply:
Oh, I can appreciate it. I'm of a similar obsession, particularly for comments, even to the point of enabling the visual "line" for my editor.

![image](https://user-images.githubusercontent.com/1779930/52652007-41d8cc80-2ebb-11e9-8533-cac0f60cc274.png)

I didn't intend to make a thing of it. My only worry might be that the indentation implies that it's of a similar level as the body of the callback, when in fact it's still part of the earlier (`it`) level.

Reply:
> I'm of a similar obsession, particularly for comments, even to the point of enabling the visual "line" for my editor.

Heh!  I do the same in my IDE + have my IDE configured to show me when line length is exceeded.